account_analytic_category/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module introduces account.analytic.category
+as a replacement for account.analytic.group which was removed in Odoo 16.0.
```

Comment:
Is there a reason to rename the model?

Reply:
Odoo called the model for [Analytic Categories](https://github.com/odoo/odoo/blob/15.0/addons/analytic/models/analytic_account.py#L37) and I thought maybe this is a better name. But I am flexible on this.